open-reac/src/main/java/com/powsybl/openreac/parameters/output/network/ShuntCompensatorNetworkOutput.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.openreac.parameters.output.network;
+
+import com.powsybl.ampl.converter.AmplSubset;
+import com.powsybl.commons.util.StringToIntMapper;
+import com.powsybl.iidm.modification.ShuntCompensatorPositionModification;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.ShuntCompensator;
+
+/**
+ * @author Nicolas Pierre <nicolas.pierre at artelys.com>
+ */
+public class ShuntCompensatorNetworkOutput extends AbstractNetworkOutput<ShuntCompensatorPositionModification> {
```

Comment:
@nicolas-pierr indeed I think we have forgotten that a shunt can be reconnected by the optimizer. And we don't have any modification for that.